cognee-mcp/apps-src/src/design.css

```
@@ -0,0 +1,591 @@
+  --cg-border-focus:     #6510F4;
+
+  /* Text */
+  --cg-fg-dark:          #18181B;
+  --cg-fg-body:          #3F3F46;
+  --cg-fg-secondary:     #52525B;
+  --cg-fg-muted:         #71717A;
+  --cg-fg-placeholder:   #A1A1AA;
+  --cg-fg-disabled:      #D4D4D8;
+
+  /* Typography (system fallback only — TWK/Parabole not bundled) */
+  --cg-font-text: "Inter", system-ui, -apple-system, BlinkMacSystemFont, sans-serif;
```

Stylelint (17.9.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)